packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -887,5 +914,77 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
         });
       });
     });
+
+    describe('@hbarlimiter-batch3 BASIC Tier', () => {
+      const accounts: AliasAccount[] = [];
+
+      beforeEach(async function () {
+        logger.info(`${requestDetails.formattedRequestId} Creating accounts`);
+        logger.info(
+          `${requestDetails.formattedRequestId} HBAR_RATE_LIMIT_TINYBAR: ${ConfigService.get(
+            'HBAR_RATE_LIMIT_TINYBAR',
+          )}`,
+        );
```

Comment:
Since this suite provides coverage when the limiter is off, there should likely be assertions confirming that `HBAR_RATE_LIMIT_TINYBAR` is set to 0 and that `isDisable` is set to true. For example, we could include an assertion like `expect(HBAR_RATE_LIMIT_TINYBAR).to.eq(0)`. If it's feasible to access `isDisable`, we could also add an assertion for that as well (not sure if the isDisable is reachable tho)

Reply:
I added ` expect(ConfigService.get('HBAR_RATE_LIMIT_TINYBAR')).to.eq(0);` to the test.  I don't think it's easily feasible to check `isEnabled()` from an acceptance test.  The running instance of the `HBarRateLimiter` would have to be captured in the server package This is covered in the unit test in the relay.